.github/workflows/test-codecov.yml

```diff
@@ -0,0 +1,21 @@
+name: Using Codecov for Native Python Connections
+on: [push]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python 3.8
```

Comment:
isn't `Python 3.7` the actual version we're using today?

Reply:
agree, fixed